src/include/migraphx/operators.hpp

```diff
@@ -1356,6 +1356,108 @@ struct undefined
     argument compute(const shape&, const std::vector<argument>&) const { return {{}, nullptr}; }
 };
 
+struct batch_contiguous
+{
+    int axis = 0;
+    std::vector<int> slice_dims;
+    std::string name() const { return "batch_contiguous"; }
+
+    shape compute_shape(std::vector<shape> inputs) const
+    {
+        auto input_shape = inputs[0];
+        std::vector<std::size_t> out_dims;
+        out_dims.push_back(input_shape.elements());
+        return {input_shape.type(), out_dims};
+    }
+
+    std::vector<uint64_t> compute_index_map(const shape& s) const
+    {
+        std::vector<uint64_t> index_map;
+        uint64_t unit_slice = 1;
+        int axis_id         = 0;
+        if((axis < 0) || (axis >= s.lens().size()))
+            MIGRAPHX_THROW("invalid split axis");
```

Comment:
Maybe we can put the operator name in the error message like: "BATCH_CONTIGUOUS: invalid spli axis", then if there is an error, we know it is from this operator.

Reply:
same suggestion for other error messages.

Reply:
done